packages/offix-datastore/src/Model.ts

```diff
@@ -17,6 +18,40 @@ export type Fields<T> = {
     [P in keyof T]: FieldOptions
 };
 
+/**
+ * Model Config options
+ */
+export interface ModelConfig<T = unknown> {
+    /**
+     * Model name
+     */
+    name: string;
+
+    /**
+     * Model store name, defualts to `user_${name}`
+     */
+    storeName?: string;
+
+    /**
+     * Model fields
+     */
+    fields: Fields<T>;
+
+    /**
+     * Delta Query and Subscription filter
+     */
+    predicate?: Predicate<T>;
```

Comment:
Delta query predicate can be different than subscription (due to subscription limitations. We do not have this well documented so just mentioning it now for the moment. Subscriptions are still not well covered in backend (yet)

Reply:
No problem. Add separate predicates for subscription and delta query